### PR TITLE
feat(pd): integrate `pd-grpc`&`pd-common`&`pd-client`

### PR DESCRIPTION
**Pull request by VGalaxies**
subtask of https://github.com/apache/incubator-hugegraph/issues/2481

merge https://github.com/apache/incubator-hugegraph/pull/2460 to master

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -86,13 +86,13 @@
     </scm>
 
     <properties>
-        <revision>1.3.0</revision>
+        <revision>1.5.0.1</revision>
```
Comment (VGalaxies) [inline pom.xml:89]: @imbajin Which version number should we use here?

File: pom.xml [hunk 2]
```diff
@@ -199,7 +199,7 @@
                                     <!-- TODO: uncomment for checking dependency conflicts -->
                                     <!-- <DependencyConvergence/> -->
                                     <requireJavaVersion>
-                                        <version>[1.8,12)</version>
+                                        <version>[11,)</version>
```
Comment (VGalaxies) [inline pom.xml:202]: @imbajin Do we enforce a specific Java version here, abandoning Java 8?

Comment (imbajin) [reply pom.xml:202]: > @imbajin Do we enforce a specific Java version here, abandoning Java 8?

In my view, Yes (we could just define it in the root pom?)

File: pom.xml (hunk 1 above)
Comment (imbajin) [reply pom.xml:89]: > @imbajin Which version number should we use here?

`1.5.0` for master (we could change it if need)

**Review by imbajin — APPROVED** [review]

**Comment by imbajin** [comment]
Note, set version to `1.5.0` before release~

**Review by JackyYangPassion — APPROVED** [review]

**Pull request merged**